kudobuilder/frameworks
# Kafka: updated to new kudo 0.3.0 version including renaming to operator

How to test:

1. Clone current master in `kudobuilder/kudo`
2. Have this branch in your `kudobuilder/operators` dir
3. Install from filesystem via:

3.a.
```bash
go run cmd/kubectl-kudo/main.go install --instance=zk /Users/fabianbaier/go/src/github.com/kudobuilder/operators/repository/zookeeper/0.1.0
```
*we should probably rename the `0.1.0` folder in zookeeper to `operator` is that is not kinda redundant. The main reason for that is just that installing from filesystem and having it in the upper level ( `repository/zookeeper` ) would fail because of the `repository/zookeeper/docs` folder.*
3.b.
```bash
go run cmd/kubectl-kudo/main.go install --instance=kafka /Users/fabianbaier/go/src/github.com/kudobuilder/operators/repository/kafka/operator
```
